extension/scripts/features/total-portfolio-value/ttTotalPortfolioValue.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+(async () => {
+	if (isFlying() || isAbroad()) return;
+
+	const feature = featureManager.registerFeature(
+		"Total Portfolio Value and Profit",
+		"stocks",
+		() => settings.pages.stocks.valueAndProfit,
+		null,
+		addProfitAndValue,
+		removeProfitAndValue,
+		{
+			storage: ["settings.pages.stocks.valueAndProfit"],
+		},
+		null
+	);
+
+	async function addProfitAndValue() {
+		await requireElement("#stockmarketroot [class*='stock___']");
+		const totalValue = [...document.findAll("[class*='stockOwned__'] [class*='value__']")]
+			.map((x) => parseInt(x.innerText.replace(/[$,]/g, "").trim()))
+			.reduce((a, b) => (a += b), 0);
```

Comment:
Instead of the reduce, use `totalSum`. It's a custom Array prototype function that does the same, but makes it clear what it actually does.